apps/docs/components/GithubCard.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
```

Comment:
Do this cards need a different background? Or are they supposed to blend with the site color?
![Screen Shot 2022-11-07 at 11 49 56](https://user-images.githubusercontent.com/1002620/200339739-8e5e1813-8f32-463d-9648-844bb58e2047.png)

Reply:
aah good catches. Updated both.